src/service/routes/auth.js

```diff
@@ -135,8 +135,12 @@ router.post('/gitAccount', async (req, res) => {
         return;
       }
 
+      const reqUser = await db.findUser(req.user.username);
+      if(username !== reqUser.username && !reqUser.admin) {
```

Comment:
Formatting issue (may need to run `npm run format`:

```suggestion
      if (username !== reqUser.username && !reqUser.admin) {
```